.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "airbnb-base",
```

Comment:
It would probably be a good idea to use a common set of lint rules in starter kit. Are these rules commonly used in the garage or a personal preference?

Reply:
This one is just a personal preference. Is there a specific set of lint rules that was used for the nodejs chaincode project? I can take a look and just grab that one since it's also part of the starter kit

Reply:
Looks like nodejs chaincode project has this https://github.com/IBM-Blockchain-Starter-Kit/nodejs-chaincode-bootstrap/blob/master/.eslintrc.js but not sure where that came from/which would be better. I don't mind which really but it would be nice to be consistent. Maybe ask on https://gitter.im/IBM-Blockchain-Starter-Kit/Lobby to see if anyone else wants to jump in ...or maybe not ;)